GitUI/BranchTreePanel/RepoObjectsTree.cs

```diff
@@ -234,22 +237,41 @@ bool IsOverride(MethodInfo m)
         }
 
         /// <summary>
-        /// Toggles filtering mode on or off to the git refs present in the left panel depending on the app's global filtering rules .
-        /// These rules include: show all branches / show current branch / show filtered branches, etc.
+        /// FormBrowse refreshing the side panel when refreshing the grid.
         /// </summary>
         /// <param name="isFiltering">
         ///  <see langword="true"/>, if the data is being filtered; otherwise <see langword="false"/>.
         /// </param>
-        /// <param name="getRefs">Function to get refs.</param>
         /// <param name="forceRefresh">Refresh may be required as references may have been changed.</param>
-        public void Refresh(bool isFiltering, bool forceRefresh, Func<RefsFilter, IReadOnlyList<IGitRef>> getRefs)
+        /// <param name="getRefs">Function to get refs.</param>
+        /// <param name="getStashRevs">Lazy accessor for stash commits.</param>
+        public void RefreshWhenLoading(bool isFiltering, bool forceRefresh, Func<RefsFilter, IReadOnlyList<IGitRef>> getRefs, Lazy<IReadOnlyCollection<GitRevision>> getStashRevs)
```

Comment:
Looks like in different parts we have different order of the same arguments:
![image](https://user-images.githubusercontent.com/4403806/196425121-ac0cba2f-2e14-45cf-a501-d98500d5b9f0.png)

Reply:
> I will not name who set the original order in the event handlers, the side panel methods or who requested changes previously in this PR. We all have a right to change our minds.

😆